Client/TeamTalkClassic/TeamTalkDlg.cpp

```diff
@@ -276,7 +276,7 @@ void CTeamTalkDlg::Disconnect()
 void CTeamTalkDlg::Login()
 {
     int cmd = TT_DoLoginEx(ttInst,
-        STR_UTF8(m_xmlSettings.GetNickname(STR_UTF8(LoadText(IDS_DEFAULTNICKNAME, DEFAULT_NICKNAME))).c_str()),
```

Comment:
Same here. Change DEFAULT_NICKNAME to blank instead of removing handling of default values.